theforeman.org/scripts/debian/execute_pbuilder.sh

```diff
@@ -6,7 +6,7 @@ echo "--Execute pdebuild"
 
 # Build the package for the OS using pbuilder
 # needs sudo as pedebuild uses loop and bind mounts
-sudo pdebuild-${os}64
+sudo pdebuild-${os}64 --hookdir ${local_hooksdir}
```

Comment:
with https://github.com/theforeman/foreman-infra/pull/1643, this would become

```
sudo REPO_SETUP=true REPO_SUITE=${os} REPO_VERSION=${version} pdebuild-${os}64
```

technically we could even template it to not need `OS` and then default `VERSION` to `nightly`

Reply:
I like being explicit. It's only scripts that call it anyway so IMHO there's no need for human conveniences.